packages/@aws-cdk/aws-stepfunctions-tasks/lib/athena/start-query-execution.ts

```diff
@@ -183,40 +183,35 @@ export class AthenaStartQueryExecution extends sfn.TaskStateBase {
    * @internal
    */
   protected _renderTask(): any {
+
+    let renderedObject = {
+      Resource: integrationResourceArn('athena', 'startQueryExecution', this.integrationPattern),
+      Parameters: {
+        QueryString: this.props.queryString,
+        ClientRequestToken: this.props.clientRequestToken,
+        QueryExecutionContext: {
+          Catalog: this.props.queryExecutionContext?.catalogName,
+          Database: this.props.queryExecutionContext?.databaseName,
+        },
+        ResultConfiguration: {
+          EncryptionConfiguration: this.renderEncryption(),
+        },
```

Comment:
let's skip the `if/else` altogether
```suggestion
          EncryptionConfiguration: this.renderEncryption(),
          OutputLocation: this.props.resultConfiguration?.outputLocation ? `s3://${this.props.resultConfiguration?.outputLocation?.bucketName}/${this.props.resultConfiguration?.outputLocation?.objectKey}/` : undefined,
        },
```